### ISSUE TITLE
Use Github API instead of git clone

### ISSUE THREAD

**Issue by madnight** [drzel/vim-repo-edit#1]
Why? Because if you want to browse big repositories like https://github.com/torvalds/linux/ then it takes a long time to clone, while it is opening within a second on https://github1s.com/torvalds/linux/

**Comment by drzel** [comment]
You're right. Good idea. Might take me a few weeks to get to it. But I'll do it. Otherwise PRs accepted!
